README.txt

```diff
@@ -163,6 +163,7 @@ ENABLE_FREETYPE     - Build with FreeType2; Allows using TTF fonts
 ENABLE_GETTEXT      - Build with Gettext; Allows using translations
 ENABLE_GLES         - Search for Open GLES headers & libraries and use them
 ENABLE_LEVELDB      - Build with LevelDB; Enables use of LevelDB map backend (faster than SQLite3)
+ENABLE_POSTGRESQL   - Build with PostgreSQL, depends on BUILD_SERVER; Enables use of PostgreSQL map backend
```

Comment:
I initially thought that it would only be used by people running it with a server, but I have removed that, when I made the other changes.